config/src/lib.rs

```diff
@@ -1771,6 +1771,10 @@ impl<'a> RemappingsProvider<'a> {
         // use auto detection for all libs
         for r in self.lib_paths.iter().map(|lib| self.root.join(lib)).flat_map(Remapping::find_many)
         {
+            // this is an additional safety check for weird auto-detected remappings
+            if ["lib/", "src/"].contains(&r.name.as_str()) {
```

Comment:
shouldn't it include `"contracts/" as well?

In the timeless-fi sample repo, `src/lib` existed. In my own private repo, I actually had `contracts/lib`

Reply:
good point, adding `contracts/` as well